src/main/config.hpp

```diff
@@ -0,0 +1,27 @@
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include <string>
+#include <vector>
+
+const char DELIMITER = ',';
+
+class Config
+{
+public:
+	Config();
+	void detect_usecase(int argc, char *argv[]);
+
+	// EXEC MONITOR
+	std::vector<int> ppid;
+	std::vector<std::string> name;
+	int n_proc; /* number of tasks to record */
+
+	void exec_monitor_parse_args_run();
```

Comment:
Why is this a public method? Do you ever plan to let other modules call this function? You should keep everything private by default and only make public what's strictly necessary.